teacher.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'person'
+
+class Teacher < Person
+  super(specialization)
+  @specialization = specialization
```

Comment:
- [ ] Please make sure you do this operation inside the initialize method and also that you receive the `specialization` variable from the constructor (Also remember that the parent constructor expects `id, age, name, parent_permission` not `specialization`.

Reply:
ok i added with super variable that include the constructor of person